main/res/values/strings.xml

```diff
@@ -1025,6 +1029,8 @@
     <string name="waypoint_duplicate">Duplicate waypoint</string>
     <string name="waypoint_duplicated">Waypoint duplicated</string>
     <string name="waypoint_copy_of">Copy of</string>
+    <string name="waypointcalc_notes">Notes:</string>
```

Comment:
I'm afraid I don't understand.

Are you suggesting the line should read:
    `<string name="waypointcalc_notes">Notes: %s</string>`

I tried putting that in and it didn't look so good :s